fennel/datasets/test_dataset.py

```diff
@@ -1259,20 +1264,21 @@ def pipeline1(cls, a: Dataset, b: Dataset):
     )
     operator_req = sync_request.operators[2]
     o = {
-        "id": "12a2088d8d7a0d265a7bd3f694fc81aa",
-        "is_root": True,
-        "pipeline_name": "pipeline1",
-        "dataset_name": "ABDataset",
-        "join": {
-            "lhs_operand_id": "A",
-            "rhs_dsref_operand_id": "B",
-            "on": {"a1": "b1"},
-            "how": 0,
-            "within_low": "259200s",
-            "within_high": "31536000s",
-        },
-        "ds_version": 1,
-    }
+	    "id":"3338ee30aac1dc899789da9fc78fa025",
+	    "isRoot":True,
+	    "pipelineName":"pipeline1",
+	    "datasetName":"ABDataset",
+	    "join":{
+	        "lhsOperandId":"A",
+	        "rhsDsrefOperandId":"B",
+	        "on":{
+	            "a1":"b1"
+	        },
+	        "withinLow":"259200s",
+	        "withinHigh":"31536000s"
+	    },
```

Comment:
why is the how missing ?  We should not modify what the protos used to generate since that is stored state on disk

Reply:
Shouldn't the test throw an error if I copied wrong json?